internal/proto/domains/v1/domains.proto

```diff
@@ -11,6 +11,7 @@ option go_package = "github.com/absmach/supermq/internal/grpc/domains/v1";
 // domains functionalities for SuperMQ services.
 service DomainsService {
   rpc DeleteUserFromDomains(DeleteUserReq) returns (DeleteUserRes) {}
+  rpc RetrieveEntity(RetrieveEntityReq) returns (RetrieveEntityRes) {}
```

Comment:
```suggestion
  rpc RetrieveEntityStatus(RetrieveEntityStatusReq) returns (RetrieveEntityStatusRes) {}
```

Because we have `RetrieveEntityReq` and `RetrieveEntityRes` in `common.proto`